swagger.yaml

```diff
@@ -1,8 +1,8 @@
 swagger: '2.0'
 info:
-  title: OSU Directory Web API
-  description: Web API to search the OSU Online Directory
-  version: "1"
+  title: Directory API
+  description: Web API to search the OSU Online Directory.
+  version: "2"
```

Comment:
We decided to implement the new version of this API with our new express-skeleton, so the version should be using `v2` since it will be used as part of the code. Also, can you change the file name to `openapi.yaml`?

Reply:
I'll make it v2, but how will you document minor version changes? Does v2.0 make more sense?